Modules/_io/bytesio.c

```diff
@@ -636,22 +636,18 @@ bytesio_iternext(bytesio *self)
 
 /*[clinic input]
 _io.BytesIO.seek
-    pos: Py_ssize_t
-    whence: int = 0
+    offset as pos: Py_ssize_t
+    whence: int(c_default='0') = io.SEEK_SET
     /
 
-Change stream position.
+Change stream position and return the new absolute position.
 
-Seek to byte offset pos relative to position indicated by whence:
-     0  Start of stream (the default).  pos should be >= 0;
-     1  Current position - pos may be negative;
-     2  End of stream - pos usually negative.
-Returns the new absolute position.
+Set the byte offset 'offset', relative to position indicated by 'whence':
```

Comment:
The list has been removed here so just finish with a full stop.
```suggestion
Set the byte offset 'offset', relative to position indicated by 'whence'.
```

Reply:
Thanks!